### PR TITLE
Render index

### PR DESCRIPTION
**Pull request by drowsy810301**
```
from utils.render_helper import render_index
```

`render_index` replace `render`

### CODE REVIEW

File: contest/views.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 from django.template import RequestContext
 from django.forms.models import model_to_dict
 
-from index.views import custom_proc
+from utils.render_helper import custom_proc
```
Comment (henryyang42) [inline contest/views.py]: This function is only used inside ``render_index``.
I think it's not necessary to import.

**Comment by drowsy810301** [comment]
fixed

**Comment by nickelcheng** [comment]
+1, @bruce3557 please check

**Comment by bruce3557** [comment]
+1, going to merge

**Pull request merged**